core/container/src/main/java/org/wildfly/swarm/container/runtime/usage/ConfigurableManagerUsageVariableSupplier.java

```diff
@@ -0,0 +1,36 @@
+package org.wildfly.swarm.container.runtime.usage;
+
+import java.util.Optional;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.wildfly.swarm.container.runtime.ConfigurableHandle;
+import org.wildfly.swarm.container.runtime.ConfigurableManager;
+
+/**
+ * Created by bob on 8/30/17.
+ */
+@ApplicationScoped
+public class ConfigurableManagerUsageVariableSupplier implements UsageVariableSupplier {
+
+    @Inject
+    public ConfigurableManagerUsageVariableSupplier(ConfigurableManager manager) {
+        this.manager = manager;
+    }
+
+    @Override
+    public Object valueOf(String name) throws Exception {
+        Optional<ConfigurableHandle> configurable = this.manager.configurables().stream()
+                .filter(e -> e.key().toString().equals(name))
+                .findFirst();
+
+        if (configurable.isPresent()) {
```

Comment:
Not quite, since that'd just swap `isPresent()` for a null check, since I then have to, if present, also call `currentValue()` which will NPE if I return null from `orElse(null)`.

Reply:
Ah yes, you're right.